front-endd/src/app/app.component.html

```diff
@@ -1,3 +1 @@
-<app-studendcrud>
-  
-</app-studendcrud>
+<router-outlet></router-outlet>
```

Comment:
please move the basic html's here... Let the components have only their component logic